src/definitions/modules/toast.js

```diff
@@ -47,68 +47,90 @@ $.fn.toast = function(parameters) {
         selector        = settings.selector,
         error           = settings.error,
         namespace       = settings.namespace,
+        fields          = settings.fields,
 
         eventNamespace  = '.' + namespace,
         moduleNamespace = namespace + '-module',
 
         $module         = $(this),
-        $toastBox       = $('<div/>',{'class':settings.className.box}),
-        $toast          = $('<div/>'),
-        $progress       = $('<div/>',{'class':settings.className.progress+' '+settings.class}),
-        $progressBar    = $('<div/>',{'class':'bar'}),
-
-        $close          = $('<i/>',{'class':'close icon'}),
+        $toastBox,
+        $toast,
+        $actions,
+        $progress,
+        $progressBar,
+        $animationObject,
+        $close,
         $context        = (settings.context)
           ? $(settings.context)
           : $('body'),
 
+        isComponent     = $module.hasClass('toast') || $module.hasClass('message'),
```

Comment:
Why it is needed to distinguish `toast` from `message` ?

`isToastComponent` may be a better name to express intent.

Reply:
Because the styling of the message component was supported before as an alternative to style toasts (and needed some css adjustments)
See https://fomantic-ui.com/modules/toast.html#use-message-style

Reply:
@exoego  changed the variable Name to `isToastComponent` as suggested while also adding support for card layout as well 😃 by 2fe990b8eb8c07c229227e3b2670eb934f1ab872